packages/types/src/payment/provider.ts

```diff
@@ -1,4 +1,33 @@
 import { PaymentSessionStatus } from "./common"
+import { CustomerDTO } from "../customer"
+import { AddressDTO } from "../address"
+import { ProviderWebhookPayload } from "./mutations"
+
+/**
+ * Normalized events from payment provider to internal payment module events.
+ */
+export enum PaymentActions {
```

Comment:
todo: enum should not be part of the type package but moved to the utils package

Reply:
what if a type definition in `types` depends on that enum? since `utils` can't be a dep of `types` we were writing duplicate declarations in some cases one in `utils` and one in `types`

Reply:
I guess we would have to create a dedicated type for the type definition in `@medusajs/types` (lot of types here :D)